packages/suite/src/views/guide/Feedback.tsx

```diff
@@ -187,6 +195,45 @@ const Feedback = ({ type }: Props) => {
         setView('GUIDE_DEFAULT');
     }, [device, firmwareType, type, setView, sendFeedback, description, category, rating?.id]);
 
+    const feedbackCategories: FeedbackCategory[] = [
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_DASHBOARD" />, value: 'dashboard' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_ACCOUNT" />, value: 'account' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_SETTINGS" />, value: 'settings' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_SEND" />, value: 'send' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_RECEIVE" />, value: 'receive' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_TRADE" />, value: 'trade' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_OTHER" />, value: 'other' },
+    ];
+
+    // Router apps does not match 1:1 to Feedback Categories
+    const getDefaultCategory = (): FeedbackCategory | undefined => {
+        const { app, route } = router;
+        const routePattern = route?.pattern || '';
+
+        if (routePattern.startsWith('/accounts/coinmarket')) {
+            return feedbackCategories.find(item => item.value === 'trade');
+        }
+        if (routePattern.startsWith('/accounts/send')) {
+            return feedbackCategories.find(item => item.value === 'send');
+        }
+        if (routePattern.startsWith('/accounts/receive')) {
+            return feedbackCategories.find(item => item.value === 'receive');
+        }
+
+        switch (app) {
+            case 'dashboard':
+                return feedbackCategories.find(item => item.value === 'dashboard');
```

Comment:
This, in combination with  the `FeedbackCategory | undefined` return type means that if we'll change the `feedbackCategories` array by eg. removing an item this will fail silently. It would be cool using an indexing that would fail at compile time when referencing a category, that doesn't exist. Like for example

Defining a `feedbackCategories` as an object:

```js
    const feedbackCategories = {
        dashboard: <Translation id="TR_FEEDBACK_CATEGORY_DASHBOARD" />,
        ...
    }
```
Then here using the explicit indexes instead of `Array.find`
```js
        switch (app) {
            case 'dashboard':
                return {value: 'dashboard', label: feedbackCategories['dashboard']}
```

And transforming the `feedbackCategories` to the React format just in time:
```tsx
<Select
                                defaultValue={defaultCategory}
                                options={feedbackCategories.entries().map([value, label] => {value, label})}

```

It's a bit more convoluted but also safer. Feel free to resolve this on your own discretion. .)

Reply:
Great solution! Implemented. (db89a0dd9caafe0428e209b0ecb834a18bbbbf8b) squashed sorry

Reply:
Cool. Thx. Resolving.